assets/js/meta-box.js

```diff
@@ -0,0 +1,33 @@
+/* global jQuery */
+( function( $ ) {
+
+	$( document ).ready( function() {
+
+		$( 'input[name="primer-layout-override"]' ).change( function() {
+
+			if ( 1 == $( this ).val() ) {
```

Comment:
Fixed in https://github.com/godaddy/wp-primer-theme/commit/d8191ca6748dfbeeb8a0bdd28e9cc8f99eca1252